README.md

```diff
@@ -1,2 +1,17 @@
-# Projecto-XSZ
-Projecto personal para la asignatura de IV
+## Problema 1:
+Para las personas que ven animes al acabar una serie es muy frecuente que no sepan que ver próximamente. Encontrar un título que se ajuste a sus gustos y que merezca la pena a veces es trabajoso.
+
+#### Propuesta 1:
+A partir de la lista de animes vistos (viendo actualmente, completado, dejado, etc) almacenada y exportada de la página MyAnimeList, recomendar a los usuarios una lista de series que les puedan interesar.
+
+## Problema 2:
+Para los fabricantes de comidas procesadas (por ejemplo, fideos instantáneos) les interesan saber antes de poner a producción un prototipo cuestiones como: cuáles son las características de los productos similares que hay en el mercado, que ingredientes utilizan en común, cuál es su precio de venta medio, cuál es el producto que más venta tiene y por qué, etc; para valorar sobre la rentabilidad del prototipo y la necesidad de realizar cambios.
+
+#### Propuesta 2:
+Recopilando información sobre un tipo de producto determinado en la web, extraer datos relevantes y generar informes sobre cualidades que influyan en su comercialización.
```

Comment:
Por favor, tratar de esforzaros un poco en definir qué es lo que vais a hacer, porque de eso es de lo que estáis hablando. "Datos" "Relevantes" "De la web". ¿Te das cuenta que si aceptara esta idea, tendrías que programarla? ¿Qué usarías para "dato"? ¿Qué para "relevante"?

Reply:
Los datos de los que hablo estoy pensando en extraerlos a partir de Web Scraping. Y lo relevante que me refiero son por ejemplo valores nutricionales medios, ingredientes en común, precio medio, etc.

Reply:
Ya imagino lo del scraping. ¿Qué vas a escrapear? ¿De donde?